RELEASES.md

```diff
@@ -138,13 +138,15 @@ for the list of actively tested versions. Kubernetes only supports n-3 minor
 release versions and containerd will ensure there is always a supported version
 of containerd for every supported version of Kubernetes.
 
-| Kubernetes Version | containerd Version | CRI Version     |
-|--------------------|--------------------|-----------------|
-| 1.24               | 1.7.0+, 1.6.4+     | v1, v1alpha2    |
-| 1.25               | 1.7.0+, 1.6.4+     | v1, v1alpha2 ** |
-| 1.26               | 1.7.0+, 1.6.15+    | v1              |
-| 1.27               | 1.7.0+, 1.6.15+    | v1              |
-| 1.28               | 1.7.0+, 1.6.15+    | v1              |
+| Kubernetes Version | containerd Versions           | CRI Version     |
+|--------------------|-------------------------------|-----------------|
+| 1.24               | 1.7.0+, 1.6.4+                | v1, v1alpha2    |
+| 1.25               | 1.7.0+, 1.6.4+                | v1, v1alpha2 ** |
+| 1.26               | 1.7.0+, 1.6.15+               | v1              |
```

Comment:
1.24 and 1.25 have reached EOL.
1.26 will reach EOL tomorrow.
https://kubernetes.io/releases/